### PR TITLE
 MNIST Digit Classification using Neural Networks

### PR DESCRIPTION
**Pull request by Cgarg9**
# Pull Request for DL-Simplified 💡
## Issue Title :  MNIST Digit Classification using Neural Networks

- **Info about the related issue (Aim of the project)** : To classify digits in the MNIST dataset using neural networks.
- **Name:** : Chirag Garg
- **GitHub ID:**: https://github.com/Cgarg9
- **Email ID:**: chiraggarg.red@gmail.com
- **Idenitfy yourself: Codepeak'23

Closes:[ (https://github.com/abhisheks008/DL-Simplified/issues/315)](https://github.com/abhisheks008/DL-Simplified/issues/315)

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
I have added two models to classify the digits. Model 1) Artificial Neural Network, Model 2) Convolutional Neural Network.
I have added two models to classify the digits. Model 1) Artificial Neural Network, Model 2) Convolutional Neural Network. These networks were trained for 10 epochs and managed to achieve about 97% and 99% accuracy respectively.
## Type of change ☑️

What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
A testing dataset is provided by MNIST and the code was tested against it.
Describe how have you verified the changes made
All errors faced have been resolved using the guidance from various online resources and official documentations of the libraries used.

## Checklist: ☑️
- [ x] My code follows the guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly wherever it was hard to understand.
- [ x] I have made corresponding changes to the documentation.
- [ x] My changes generate no new warnings.
- [ x] I have added things that prove my fix is effective or that my feature works.
- [ x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — COMMENTED** [review]
Why you have created the `Digit Classification` folder?

**Comment by Cgarg9** [comment]
Sorry i created that earlier for this project. I'll delete it and make a new pr. any more changes??

**Review by abhisheks008 — APPROVED** [review]
Nice work. Your PR is approved and ready to be merged.
@Cgarg9
